### DISCUSSION TITLE
Install Repo Issue

### DISCUSSION THREAD

**Discussion by srlandon** [sipcapture/homer#537] [Q&A]
Trying to perform a new manual install. The script to add the repo seems to have completed successfully but i get the following when i try to install.

sudo yum install heplify-server
Last metadata expiration check: 0:00:15 ago on Thu 21 Apr 2022 09:51:18 AM EDT.
No match for argument: heplify-server
Error: Unable to find a match: heplify-server

**Comment by lmangani** [comment]
Hello @srlandon please provide the OS details and the exact procedure and output for the steps ahead of the above. I would guess the repository installation failed somehow.

**Reply by srlandon** [reply]
NAME="Oracle Linux Server"
VERSION="8.5"
ID="ol"
ID_LIKE="fedora"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="8.5"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Oracle Linux Server 8.5"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:oracle:linux:8:5:server"
HOME_URL="https://linux.oracle.com/"
BUG_REPORT_URL="https://bugzilla.oracle.com/"

ORACLE_BUGZILLA_PRODUCT="Oracle Linux 8"
ORACLE_BUGZILLA_PRODUCT_VERSION=8.5
ORACLE_SUPPORT_PRODUCT="Oracle Linux"
ORACLE_SUPPORT_PRODUCT_VERSION=8.5

 curl -s https://packagecloud.io/install/repositories/qxip/sipcapture/script.rpm.sh | sudo bash
Detected operating system as ol/8.
Checking for curl...
Detected curl...
Downloading repository file: https://packagecloud.io/install/repositories/qxip/sipcapture/config_file.repo?os=ol&dist=8&source=script
done.
Installing yum-utils...
qxip_sipcapture-source                                                                                                                                           518  B/s | 833  B     00:01
Package yum-utils-4.0.21-4.0.1.el8_5.noarch is already installed.
Dependencies resolved.
Nothing to do.
Complete!
Generating yum cache for qxip_sipcapture...
Generating yum cache for qxip_sipcapture-source...

The repository is setup! You can now install packages.

sudo yum install heplify-server
Last metadata expiration check: 0:00:39 ago on Thu 21 Apr 2022 12:25:25 PM EDT.
No match for argument: heplify-server
Error: Unable to find a match: heplify-server

**Comment by lmangani** [comment]
Sorry, Oracle Linux 8 repos are not directly supported by the script and CentOS RPM packages should be used instead.
The script will have deployed and additional yum repo config - edit it to point to [el/8] and it should work fine.
